OpenEdXMobile/res/layout/activity_single_fragment_base.xml

```diff
@@ -6,65 +6,70 @@
     android:background="@color/grey_act_background"
     android:orientation="vertical"
     android:splitMotionEvents="false"
-    android:fitsSystemWindows="true"
-    >
+    android:fitsSystemWindows="true">
```

Comment:
Sounds good. I wasn't sure since there was only one edge of the system window at the top.